api/v11/drawingTemplates.py

```diff
@@ -74,6 +75,9 @@ def removeTemplate(clientId, userId, tool, name):
 
 
 def createOrUpdateTemplate(clientId, userId, name, tool, content):
+	if not content
+		return common.error('No content to save')
+
```

Comment:
Also we should check for `name` emptiness.
The actual problem is that we cannot remove a drawing template by its empty name.

These checks are helpful anyway, but looks that we should allow to delete a template with empty name here

https://github.com/tradingview/saveload_backend/blob/64b6bf89f413dacd293a261a97e59c7fcd048b6b/api/v11/drawingTemplates.py#L34-L41

Then probably we can revert changes in  `getTemplates`